components/conversations/Comment.js

```diff
@@ -13,25 +15,62 @@ import Avatar from '../Avatar';
 import ConfirmationModal from '../ConfirmationModal';
 import Container from '../Container';
 import { Box, Flex } from '../Grid';
-import Hide from '../Hide';
 import HTMLContent from '../HTMLContent';
+import RoundHoriztonalDotsIcon from '../icons/RoundHorizontalDotsIcon';
 import InlineEditField from '../InlineEditField';
 import LinkCollective from '../LinkCollective';
 import MessageBox from '../MessageBox';
 import RichTextEditor from '../RichTextEditor';
 import StyledButton from '../StyledButton';
+import StyledHr from '../StyledHr';
+import { fadeIn } from '../StyledKeyframes';
 import { P } from '../Text';
 
 import { CommentFieldsFragment } from './graphql';
 
 const CommentBtn = styled(StyledButton)`
-  height: 32px;
-  padding: 4px 16px;
+  padding: 3px 5px;
+  margin: 5px 0;
+  width: 100%;
+  text-align: left;
+  border: none;
+
   span {
-    margin-left: 0.5em;
-    @media (max-width: 52em) {
-      display: none;
-    }
+    margin-left: 12px;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 21px;
+    letter-spacing: -0.1px;
+  }
+`;
+
+const AdminActionsPopupContainer = styled(Flex)`
+  flex-direction: column;
+  background: ${themeGet('colors.white.full')};
+  border: 1px solid rgba(49, 50, 51, 0.1);
+  border-radius: 8px;
+  box-shadow: 0px 4px 8px rgba(20, 20, 20, 0.16);
+  width: 184px;
+  padding: 16px;
+  animation: ${fadeIn} 0.3s ease-in-out;
```

Comment:
This creates a small delay before you can use the menu, I'm not a fan of it. We'll see with design if they later want to add an animation for this

```suggestion
  animation: ${fadeIn} 0.3s ease-in-out;
```